src/client/eppo-client.ts

```diff
@@ -162,12 +166,15 @@ export type FlagConfigurationRequestParameters = {
 };
 
 export default class EppoClient implements IEppoClient {
-  private queuedEvents: IAssignmentEvent[] = [];
+  private queuedAssignmentEvents: IAssignmentEvent[] = [];
   private assignmentLogger?: IAssignmentLogger;
+  private queuedBanditEvents: IBanditEvent[] = [];
+  private banditLogger?: IBanditLogger;
   private isGracefulFailureMode = true;
   private assignmentCache?: AssignmentCache;
   private requestPoller?: IPoller;
   private evaluator = new Evaluator();
+  private banditEvaluator = new BanditEvaluator();
```

Comment:
nit: make fields readonly if you can - we need to setup an eslint rule for that

Reply:
oh good call